cmd/atlas/internal/cloudapi/client.go

```diff
@@ -316,6 +323,7 @@ type (
 	roundTripper struct {
 		token        string
 		extraHeaders map[string]string
+		transport    http.RoundTripper
```

Comment:
usuaslly this one is called `base`.
```suggestion
		base    http.RoundTripper
```